.github/workflows/greetings.yml

```
@@ -0,0 +1,13 @@
+name: Greetings
+
+on: [pull_request, issues]
+
+jobs:
+    greeting:
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/first-interaction@v1
+                with:
+                    repo-token: ${{ secrets.GITHUB_TOKEN }}
+                    issue-message: '**Thanks for opening your first issue here!**<br/><br/>Be sure to follow the issue template! Your issue will be reviewed by a maintainer and labeled accordingly for further actions.'
```

```suggestion
                    issue-message: '**Thanks for opening your first issue here!**<br/><br/>Be sure to follow the issue template. Your issue will be reviewed by a maintainer and labeled for further action.'
```

Was a little bit too excited before ;)

---

Thanks! Just force pushed the changes.